### DISCUSSION TITLE
Welcome to mitmproxy Discussions!

### DISCUSSION THREAD

**Discussion by mhils** [mitmproxy/mitmproxy#4903] [General]
# Welcome to the mitmproxy Discussions forum

This discussion forum is a place for users and developers to discuss mitmproxy. Please feel free to

  * Ask questions about mitmproxy
  * Request features
  * Share ideas / demos / interesting finds
  * Engage with other community members.
  * Welcome others. Remember that this is a community we build together!

We used to refer people to StackOverflow for this, but that splits the community over multiple sites and makes search tricky. We'll now try GitHub Discussions and hope you all like it! 😃 

## Expectations

Mitmproxy is developed by a bunch of volunteers. The good news is we're not a closed group, everyone is welcome to participate. The bad news is we don't have dedicated support staff that will help with all issues, so your particularly tricky questions may not find an answer right away. We encourage everyone to help contribute to the discussions.

## Reporting bugs

If you would like to report a bug, please use the [Issues tab](https://github.com/mitmproxy/mitmproxy/issues/new/choose)!

## Show and Tell

We frequently have people reach out in private and share some really cool and creative ways they were using mitmproxy. We want to continue this trend and bring a bit more visibility to it. Feel free to share how you're using mitmproxy with screenshots or videos!

**Comment by xmo-odoo** [comment]
Nothing to ask (yet?), just wanted to thank you guys for mitmproxy, excellent tool for mocking / faking services when configuring the endpoint(s) is impossible, nonsensical, or undesirable. In that case, the interception is just a `mitmdump --map-remote` and a few envvars away. Just stellar work.

**Comment by soovorow** [comment]
Hi there! I wonder if there is a way to sniff wss on iOS? It's works perfectly with HTTP, HTTPS requests. Also WS and WSS requests in browser only and socks5 mode). But for other apps WS/WSS does not appear. Is it possible to see WSS traffic from iOS apps except web browser? What should i do to achieve it? Thanks.

**Comment by jannielove** [comment]
Can this be used as a proxy in Telegram?

**Comment by netman32** [comment]
Hello Mitmproxy team!

I would like to ask you three questions:
1. Why is mitmproxy in transparent mode displaying/catching only HTTP traffic? I don see any HTTPS decoded...
2. How can I save all event messages?
3. How can I verify that mitm CA cert is installed on my machine?

My test setup is as follow: I run downloaded version of mitmproxy in transparency mode. The configuration is as per the web documentation, namely on Linux machine I configured:
 - Enabling if IP forwarding
echo 1 > /proc/sys/net/ipv4/ip_forward
echo 1 > /proc/sys/net/ipv6/conf/all/forwarding

- Disabled ICMP redirect
echo 0 > /proc/sys/net/ipv4/conf/all/send_redirects

- Configured IP tables rules to redirect traffic to mitmproxy
sudo iptables -t nat -F
sudo iptables -t nat -A PREROUTING -p tcp --dport 80 -j REDIRECT --to- port 8080 
sudo iptables -t nat -A PREROUTING -p tcp --dport 443 -j REDIRECT --to- port 8080 
sudo ip6tables -t nat -A PREROUTING -p tcp --dport 80 -j REDIRECT --to- port 8080
sudo ip6tables -t nat -A PREROUTING -p tcp --dport 443 -j REDIRECT -- to-port 8080

- I put my Linux machine between the target host (my mobile) and the router of the network. I test in WiFi network without password

At the end on the mitmproxy I see only http traffic. 
In the event log I see the following error "error establishing server connection: [Error 110] Connect Call Failed".

The idea is to see HTTPS traffic from the test target mobile without manipulating the mobile in anyway.

With wireshark I see that traffic is coming to my Linux machine. 

Command:
sudo netstat -plnut shows ESTABLISHED connections to mitmproxy.

Any idea what can be wrong?

Many thanks in advance!

**Comment by rohit3607** [comment]
Can anyone here find me terabox api 
I'll pay

**Comment by SMARTCODING-eng** [comment]
Can I use it to make Facebook requests?
